src/index.html

```diff
@@ -10,13 +10,101 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
+
     <title>Moyo header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
-  <body>
-    <h1>Moyo header</h1>
-  </body>
+  <header class="header">
+    <img
+      src="images/logo.png"
+      alt="Логотип"
+      class="logo_img"
+    />
+    <nav class="nav">
+      <ul class="nav_list">
+        <li class="nav_item">
+          <a
+            class="nav_link is-active"
+            href="./index.html"
```

Comment:
All navigation links currently point to `./index.html`. Ensure that each link points to the correct destination page if different pages are intended.